R/match-standards-chunks.R

```diff
@@ -235,55 +238,3 @@ pandoc.table(tmp[, c("mzmed", "ppm_error", "rtmed", "target_RT",
                      "mean_high", "mean_low")], style = "rmarkdown",
              split.tables = Inf,
              caption = paste0("Feature to standards matches for ", std))
-
-
-## ---- plot-ms2-hmdb-heatmap ----
```

Comment:
@andreavicini, I realized that we can't re-use code blocks for figures: they will always overwrite previously generated images (since each code block has then the same name). I converted the code here into a helper function and call that instead.

Reply:
I noticed that too but I couldn't understand why it happened (and thought I only had that problem). Thank you!